src/config/global-config.c

```diff
@@ -56,6 +58,29 @@ int Read_GlobalSK(XML_NODE node, void *configp, __attribute__((unused)) void *ma
                 merror(XML_VALUEERR, node[i]->element, node[i]->content);
```

Comment:
Under `Config->syscheck_auto_ignore = 0;`, add the same default values for the `auto_ignore` option. This will set these default values if the user does not add the attribute options (likely in upgraded configurations).